x/distribution/client/cli/query.go

```diff
@@ -6,16 +6,36 @@ import (
 	"strings"
 
 	"github.com/spf13/cobra"
+	amino "github.com/tendermint/go-amino"
 
+	"github.com/cosmos/cosmos-sdk/client"
 	"github.com/cosmos/cosmos-sdk/client/context"
 	"github.com/cosmos/cosmos-sdk/codec"
 	sdk "github.com/cosmos/cosmos-sdk/types"
 	"github.com/cosmos/cosmos-sdk/version"
-	distr "github.com/cosmos/cosmos-sdk/x/distribution"
 	"github.com/cosmos/cosmos-sdk/x/distribution/client/common"
 	"github.com/cosmos/cosmos-sdk/x/distribution/types"
 )
 
+// GetQueryCmd returns the cli query commands for this module
+func GetQueryCmd(storeKey string, cdc *amino.Codec) *cobra.Command {
+	distQueryCmd := &cobra.Command{
+		Use:   "distr",
```

Comment:
This `distr` should be a variable somewhere ideally.

Reply:
sure, we could use the existing `ModuleName` I've updated for both tx and query cli

Reply:
exactly what I was thinking! Great deal.